### PR TITLE
Adorable command

### PR DESCRIPTION
**Pull request by DoctorVWA**

### CODE REVIEW

**Comment by pedrofracassi** [comment]
What if the user doesn't pass any args? In my opinion the command could generate an adorable avatar based on the user's ID or tag.

File: src/commands/memes/adorable.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('../../')
+
+module.exports = class Adorable extends Command {
+  constructor (client) {
+    super(client)
+
+    this.name = 'adorable'
+    this.aliases = ['adrlb']
```
Comment (pedrofracassi) [inline src/commands/memes/adorable.js]: Is this really necessary?

**Comment by pedrofracassi** [comment]
Please move it to `commands/misc`

**Comment by bolsomito** [comment]
Everything seems to be in conformity with everything. Merging with the `master` branch.

**Pull request merged**